### PR TITLE
fix ArgumentNullException

### PR DESCRIPTION
**Pull request by surebrec**
fix `ArgumentNullException `in `string.Join("; ", list)`, when `list == null`

### CODE REVIEW

**Comment by Starwer** [comment]
The change is correct, fix the bug without side-effects. Thanks a lot for this fix !

File: src/TagLib/Matroska/Tag.cs [hunk 1]
```diff
@@ -384,6 +384,7 @@ public string[] Get(string key, string subkey = null, bool recu = true)
 						{
 							str = null;
 						}
+						else
```
Comment (decriptor) [inline src/TagLib/Matroska/Tag.cs]: Could you put this else block in `{` `}`?

**Comment by decriptor** [comment]
It is a simple fix, but if you are able to add a unit test for this that'd be awesome :)

File: src/TagLib/Matroska/Tag.cs (hunk 1 above)
Comment (Starwer) [reply src/TagLib/Matroska/Tag.cs]: the `else` block conditions the next `if`. IMHO, a better cosmetic fix would be the put the `if` on the same line to get an `else if` construct.

Comment (decriptor) [reply src/TagLib/Matroska/Tag.cs]: I'm a fan of whatever makes the code more readable. If it is an `else if` then agreed they need to be on the same line :)

**Pull request merged**